apps/condo/domains/common/hooks/useMultipleFiltersModal.tsx

```diff
@@ -455,22 +470,28 @@ const Modal: React.FC<MultipleFiltersModalProps> = ({
             labelCol={LABEL_COL_PROPS}
             initialValue={get(selectedFiltersTemplate, 'name')}
             required
+            rules={[{ required: true, message: FieldRequiredMessage, whitespace: true }]}
         >
             <Input placeholder={NewTemplatePlaceholder} />
         </Form.Item>
-    ), [NewTemplatePlaceholder, TemplateLabel, selectedFiltersTemplate])
+    ), [FieldRequiredMessage, NewTemplatePlaceholder, TemplateLabel, selectedFiltersTemplate])
 
     const NewFiltersTemplateNameInput = useCallback(() => (
         <Form.Item
             name='newTemplateName'
             label={NewTemplateLabel}
             labelCol={LABEL_COL_PROPS}
+            rules={[{ required: false, message: FieldRequiredMessage, whitespace: true }]}
```

Comment:
this array will be recreated on each render and have new address in memory

Reply:
done